worklets/Overview.bs

```diff
@@ -247,6 +246,41 @@ When a user agent is to <dfn>set up a worklet environment settings object</dfn>,
 
 Issue: Merge this with https://html.spec.whatwg.org/multipage/workers.html#set-up-a-worker-environment-settings-object
 
+### Fetching a module worklet script tree ### {#fetching-a-module-worklet-script-tree}
+
+When a user agent is to <dfn>fetch a module worklet script tree</dfn>, given |resolvedModuleURL|,
+|outsideSettings|, and |insideSettings| it must run the following steps:
+
+    1. <a>Fetch a module script tree</a> given |resolvedModuleURL|, |outsideSettings|, "script" the
+        empty string (as no cryptographic nonce is present for worklets), "not parser-inserted", "omit",
+        and |insideSettings|.
```

Comment:
Changed. I was discussing this with @mikewest (as destination is just used for the CSP check right?) settled on "script-src" being ok initially.

Let me know if I'm off the track here in what this param does. (traced it through the other day... i think)